lib/jazzy/doc_builder.rb

```diff
@@ -192,6 +190,7 @@ def self.document_index(source_module, path_to_root)
       doc[:github_url] = source_module.github_url.to_s
       doc[:dash_url] = source_module.dash_url
       doc[:path_to_root] = path_to_root
+      doc[:hide_name] = true
```

Comment:
Hides the name in the index (readme). Otherwise it looks pretty silly:

![image](https://cloud.githubusercontent.com/assets/474794/7010425/e674a708-dc55-11e4-8939-8e592a8f38cb.png)

Same for Realm.